src/components/preview.tsx

```diff
@@ -249,6 +249,10 @@ export const Preview: Component<Props> = (props) => {
     setIframeHeight(percentage);
   };
 
+  createEffect(() => {
+    localStorage.setItem('uiTheme', props.isDark ? '"dark"' : '');
```

Comment:
Thinking that maybe we should make this `props.isDark ? '"dark"' : '"default"'`, because I think the options are `"dark"`, `"default"`, and `"systemPreferred"`, with the default being systemPreferred.